doc/source/whatsnew/v0.18.0.txt

```diff
@@ -40,6 +40,7 @@ Other enhancements
 - ``read_excel`` now supports s3 urls of the format ``s3://bucketname/filename`` (:issue:`11447`)
 - A simple version of ``Panel.round()`` is now implemented (:issue:`11763`)
 - For Python 3.x, ``round(DataFrame)``, ``round(Series)``, ``round(Panel)`` will work (:issue:`11763`)
+- sys.getsizeof(obj) returns the memory usage of the object including (non-object) values (:issue:`11597`)
```

Comment:
this is not True at all. that's the purpose of ``deep``. This just allows ``sys.getsizeof(pandas_object)`` to work